alacritty_terminal/src/input.rs

```diff
@@ -455,7 +455,11 @@ impl<'a, A: ActionContext + 'a> Processor<'a, A> {
             if let Some(url) =
                 self.ctx.terminal().urls().drain(..).find(|url| url.contains(buffer_point))
             {
-                return MouseState::Url(url);
+                if self.ctx.selection_is_empty() &&
```

Comment:
I think we can move this check to the other `if` block, the one before the `if let` binding. We don't need access to the `url` so it should be easier to check it early in one block.